sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredential.java

```diff
@@ -90,8 +90,7 @@
                             + " To fix, authenticate through tooling supporting azure developer sign on."));
                     } else {
                         return Mono.error(new RuntimeException(String.format("User account '%s' was not found in the "
-                            + "shared token cache. Discovered Accounts: [ '%s' ]", username,accounts.values().stream()
-                            .map(IAccount::username).collect(Collectors.joining(", ")))));
+                            + "shared token cache. There are no discovered accounts.", username)));
```

Comment:
This is what Jianghao pointed out in Teams

Reply:
Yup. And he approved the fix.